spec/helpers/container_summary_helper_spec.rb

```diff
@@ -1,6 +1,6 @@
 describe ContainerSummaryHelper do
-  REL_HASH_WITH_LINK = [:label, :image, :value, :link, :title]
-  REL_HASH_WITHOUT_LINK = [:label, :image, :value]
+  REL_HASH_WITH_LINK = [:label, :image, :value, :link, :title].freeze
+  REL_HASH_WITHOUT_LINK = [:label, :image, :value].freeze
```

Comment:
FWIW I think these definitions get scoped to the RSpec class definition. 

`::SomeRSpecJunk::ContainerSummaryHelper::REL_HASH_WITH_LINK`